File: PYTHON/Cap07/c07ex06.py
# c07ex06.py

def fatorial(n):
  fat = 1
  for i in range(1, n+1):
    fat = fat * i
  return fat

print("Qual fatorial: ", end="")
limite = int(input())

print(f"Fatorial = {fatorial(limite)}")

input("\nTecle <Enter> para encerrar... ")

File: DART/Cap07/c07ex06.dart
// c07ex06.dart

import 'dart:io';

int fatorial(int N) {
  int fat = 1;
  for (int i = 1; i <= N; i++)
    fat *= i;
  return fat;
}

void main() {
  int limite;

  stdout.write("Qual fatorial: ");
  limite = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  print("Fatorial = ${fatorial(limite)}");

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
